applications/DropletDynamicsApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -0,0 +1,60 @@
+//    |  /           |
```

Comment:
Are you planning to have custom strategies in this new app? If no, this should be removed.

Reply:
Not at the moment. I'll remove it.